lib/engine.py

```diff
@@ -707,8 +707,8 @@ def filename_from_url(url):
 
 def gen_from_source(source):
     """Returns a generator from a source tuple.
-    
-    Source tuples are of the form (callable, args) where callable(*args)
+
+    Source tuples are of the form (callable, args) where callable(\*args)
```

Comment:
So zero build warnings?

On Aug 14, 2016 11:17 PM, "Henry Senyondo" <notifications@github.com> wrote:

In lib/engine.py
<https://github.com/weecology/retriever/pull/603#discussion_r74705844>:

> @@ -707,8 +707,8 @@ def filename_from_url(url):
>
>  def gen_from_source(source):
>      """Returns a generator from a source tuple.
> -
> -    Source tuples are of the form (callable, args) where callable(*args)
> +
> +    Source tuples are of the form (callable, args) where callable(\*args)

restructuredtext escape character

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/weecology/retriever/pull/603/files/ec8bae81bb090a822fcca7b5160bf65fd3eb0b5f#r74705844>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AGplhE3XyuDza7-yvx6JY45Jyj9nmu8eks5qf1SXgaJpZM4Jj8gG>
.